### PR TITLE
Update integrations

### PR DESCRIPTION
**Pull request by bt90**
* add Watchtower and Mailrise as official integrations
* added new section for projects integrating via JSON publishing
* ~~sorted all entries~~

### CODE REVIEW

**Comment by binwiederhier** [comment]
Sorry for not responding earlier. Now it's out of date and in conflict.

Can you just add the entries that you wanted to add, and I'll do the sorting when I have time. It's impossible to review like this. It's too much.

**Comment by bt90** [comment]
@binwiederhier fixed 🙂

**Comment by bt90** [comment]
@binwiederhier

**Comment by binwiederhier** [comment]
Sorry for the delay again. So I finally had time to review those suggestions, and normally it's a nobrainer to merge stuff into this list. But for your suggestions, I cannot find a single one of these that actually integrates with ntfy. I browsed their websites and searched Google for "jellyfin ntfy" and such. Nada. Am I missing something?

**Comment by bt90** [comment]
Sorry for the confusion. I specifically added them to a separate section because these projects prefer to use a generic JSON structure for publishing notifications. I tried to link their documentation where possible and linked the section title to https://docs.ntfy.sh/publish/#publish-as-json

While these integrations don't have the  "officially supports  ntfy" status, it can be easily set up and is the prefered way of integrating ntfy from the standpoint of these projects.

And i have a single link to post on Reddit, Mastodon etc without having to explain that "xy works perfectly fine, you just need zo..."

**Comment by binwiederhier** [comment]
I don't think I feel comfortable saying things are "officially supported" if they do not have the ntfy name somewhere on their website or in the product. Because that would mean that I could add all the things that can post HTTP requests.

I think the "Examples" page is perfect for this stuff. Though you'd need more than just listing the names of the project. I'd love to accept more examples.

File: docs/integrations.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ and uptime of third party servers, so use of each server is **at your own discre
 - [Platypush](https://docs.platypush.tech/platypush/plugins/ntfy.html) - Automation platform aimed to run on any device that can run Python
 - [diun](https://crazymax.dev/diun/) - Docker Image Update Notifier
 - [Cloudron](https://www.cloudron.io/store/sh.ntfy.cloudronapp.html) - Platform that makes it easy to manage web apps on your server
+- [Watchtower](https://containrrr.dev/watchtower/) ⭐ - A container-based solution for automating Docker container base image updates
+- [Mailrise](https://github.com/YoRyan/mailrise) - An SMTP gateway for Apprise notifications
+
+## [JSON publishing](publish.md#publish-as-json) integrations
+- [Jellyfin](https://jellyfin.org/) ⭐ - The Free Software Media System
+- [Overseer](https://docs.overseerr.dev/using-overseerr/notifications/webhooks) ⭐ - a request management and media discovery tool built to work with your existing Plex ecosystem
+- [Tautulli](https://github.com/Tautulli/Tautulli/wiki/Notification-Agents-Guide#webhook) ⭐ - A Python based monitoring and tracking tool for Plex Media Server
```
Comment (binwiederhier) [inline docs/integrations.md:47]: I checked and none of these provide official ntfy support. They can be easily integrated via HTTP or SMTP, but official to me means that the word "ntfy" appears in the code/docs/UI. And that's not true. 

If these integrate well, we can make a section, "Easily integrated via HTTP/SMTP" and link to the examples page too if there are examples. I'll do that.

**Pull request merged**